homeassistant/helpers/entity.py

```diff
@@ -207,13 +207,6 @@ def async_set_context(self, context):
 
         This method must be run in the event loop.
         """
-        if self.hass is None:
```

Comment:
These checks need to stay here before anything else happens.

Reply:
That is correct. It would be only hard to read if you did not look the whole file. The GitHub review view is some times not objective without seeing all lines :/

Reply:
Fixed.